commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/ftp/FtpFileObject.java

```diff
@@ -582,7 +582,7 @@ void abort() throws IOException {
         protected void onClose() throws IOException {
             final boolean ok;
             try {
-                ok = client.completePendingCommand();
+                ok = client.completePendingCommand() || client.getReplyCode() == 426;
```

Comment:
Whether or not this fix is right, you should use the constant ```FTPReply.TRANSFER_ABORTED```, not a magic number.

Reply:
Also, are there any other return codes that should be considered? Should a list of such RCs be configurable? That last part seems over the top.

Reply:
Thanks for the comments! I've updated the PR code to use the constant instead.

As for the second part - I'm still not sure whether other return codes are possible in "normal" circumstances. I'll be debugging in the next few days. But I think this is enough for now, no need to over-design this from the start. If there are other cases like this one, we could always make more changes later.